modules/component/src/componentNext.ts

```diff
@@ -251,7 +314,41 @@ export class ComponentNext<P1 extends ComponentProps> {
       this._comparator
     )
   }
-
+  /**
+   * Adds presentation logic to the component
+   * 1. Create view based on props and state
```

Comment:
Here I like the interspersed explanation and code-snippet... makes it easier to understand 👍